src/backend/modules/user/resolvers.js

```diff
@@ -14,8 +14,10 @@ const users = {
 };
 
 import { User } from "../../models";
+import config from "../../../../config/default.json";

```

Comment:
Instead of specifying a secret key in this `config/default.json` file, we might want to actually have the secret key stored as a `.env` file (so people looking at our code on Github won't know what the secret key is.) For example, we could add `JWT_SECRET` to our `.env` file, which doesn't get pushed up to version control.

You can check how `MONGODB_URI` is accessed in `ssr-server.js`, and do something similar for this!